src/test/java/games/strategy/engine/framework/ArgParserTest.java

```diff
@@ -75,6 +76,15 @@ public void singleUrlArgIsAssumedToBeMapDownloadProperty() {
   }
 
   @Test
+  public void singleUrlArgIsUrlDecoded() throws UnsupportedEncodingException {
```

Comment:
The exception specification is no longer necessary after your most-recent commit.